app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -193,96 +221,173 @@ You will not be eligible for the IRP, because you have spent more than 90 days i
+Before you apply for the first and second instalments of the IRP, you’ll need to check you meet the eligibility requirements.
 
-In order to be paid the IRP, you must have a UK bank account in your name (this can be digital). The IRP cannot be paid into a building society account.
+Applications are open from 30 September 2024 to end of March 2025.
+
+If you start your teaching job between 1 March 2024 and 28 February 2025 you should apply between 30 September 2024 and the end of March 2025. 
+
+If you are eligible, you should receive the money following successful completion of our eligibility checks. You must apply between 30 September 2024 and the end of March 2025 to remain eligible.
+
+
+## DfE eligibility requirements for IRP instalments
+
+### First instalment of the IRP
+
+DfE will check you meet the eligibility requirements for the first instalment of IRP by validating your application with:
+
+* [UK Visas and Immigration](https://www.gov.uk/government/organisations/uk-visas-and-immigration) 
+
+* the school where you are employed
+
+If your eligibility checks are successful, you will get the first instalment of £5,000. 
+
+### Second instalment of the IRP
+
+At the start of your second year of employment as a teacher, apply for the second instalment of the IRP. 
+
+To get the second instalment of the IRP, you will need to have:
+
+* made a successful claim for the first instalment
+
+* completed 3 consecutive terms of employment as a teacher
+
+Applications for the second instalment are open from September 2025 to March 2026.
+
+The Department for Education will validate your application with [UK Visas and Immigration](https://www.gov.uk/government/organisations/uk-visas-and-immigration) and the school or schools where you have worked since applying for the first instalment of the IRP. If you meet the eligibility requirements for the second instalment set out on this page, you should receive the second instalment of £5,000 following successful completion of our eligibility checks. 
+
+### IRP application deadlines
+
+If you take up your first teaching job in England after March 2025 and miss the deadline to apply for the first instalment of the IRP, you can apply when applications reopen in autumn 2025.
+
+If you complete 3 terms of employment as a teacher after March 2026, missing the deadline for the second instalment of the IRP, you can apply when applications reopen in autumn 2026.
+
+DfE cannot process late applications for the IRP. If you fail to apply during the correct time period, DfE will not process your application. 
+
+
+## Receiving the IRP payments
+
+You will not be paid your first instalment of the IRP before you start your teaching job. This is because eligibility requirement checks can only be completed after you’ve arrived in England.
```

Comment:
```suggestion
You can only apply for your first instalment of the IRP when you start your teaching job. This is because eligibility requirement checks can only be completed after you’ve arrived in England.
```